tests/sub_port_interfaces/sub_ports_helpers.py

```diff
@@ -771,6 +771,7 @@ def get_port(duthost, ptfhost, interface_num, port_type, ports_to_exclude=None,
         ptf_ports[port_id] = ptf_ports_available_in_topo[port_id]
 
     if 'port_in_lag' in port_type:
+        remove_acl_table(duthost, config_port_indices)
```

Comment:
Does this configuration change need to be restored after testing is done?

Reply:
Hi @wangxin. This configuration is restored per test case:
https://github.com/Azure/sonic-mgmt/blob/master/tests/sub_port_interfaces/conftest.py#L537
Also, there is reload of the configuration after the test suite:
https://github.com/Azure/sonic-mgmt/blob/master/tests/sub_port_interfaces/conftest.py#L568-L577